subsys/logging/log_backend_uart.c

```diff
@@ -31,7 +31,12 @@ LOG_OUTPUT_DEFINE(log_output, char_out, &buf, 1);
 static void put(const struct log_backend *const backend,
 		struct log_msg *msg)
 {
-	log_backend_std_put(&log_output, 0, msg);
+	u32_t flag = 0;
+
+#ifdef CONFIG_LOG_BACKEND_UART_SYST_ENABLE
```

Comment:
you could use `IS_ENABLED()`
something like
`u32_t flag = IS_ENABLED(SYST_ENABLE) ? LOG_OUTPUT_FLAG_FORMAT_SYST : 0;`
would look more concise, imo.

Reply:
Thanks, understand the IS_ENABLED